app/code/Magento/Analytics/Model/ReportXml/ModuleIterator.php

```diff
@@ -34,7 +34,7 @@ public function __construct(
      *
      * @return array
      */
-    public function current()
+    public function current(): array
```

Comment:
this is a backward-incompatible change.
I think we need to add `#[ReturnTypeWillChange]` instead, like that:
```suggestion
    #[\ReturnTypeWillChange]
    public function current()
```

Reply:
Hello @ihor-sviziev 
This is really true.
This is big BIC.
We fixed this changes.
@karyna-tsymbal-atwix thank you for the fast fix.